docs/src/_templates/versions.html

```diff
@@ -15,17 +15,18 @@
     </div>
     <script>
         console.log('script')
+        var pathname = window.location.pathname.split('padl/')[1].split('/');
+        var currentVersion = pathname[1];
+        var versionsFile = window.location.pathname.split('padl/')[0] + 'padl/docs/versions.json';
         httpRequest = new XMLHttpRequest();
         httpRequest.onreadystatechange = addVersions;
-        httpRequest.open('GET', '../versions.json');
+        httpRequest.open('GET', versionsFile);
```

Comment:
@wuhu I'm not super confident with javascript so I'd appreciate if you could take a look at the changes I made here. I made them with the intent that the `versions.json` file will always be found no matter how far down you progress in the html directories.